samples/micrometer-samples-core/build.gradle

```diff
@@ -4,15 +4,16 @@ dependencies {
     compile('ch.qos.logback:logback-classic:1.2.+') {
         force = true
     }
-    compile('org.slf4j:slf4j-api:1.7.25') {
+    compile('org.slf4j:slf4j-api:1.7.5') {
```

Comment:
should it be part of this PR?

Reply:
This is because locks were updated. It should be fine. @jkschneider can comment on that.

Reply:
@dhaval24 I have the same question on this with @SergeyKanzhelev . This looks irrelevant to this PR.